vm/vm_test.go

```diff
@@ -59,12 +59,12 @@ func TestBlockCache(t *testing.T) {
 	}
 
 	// Init metrics (called in [Accepted])
-	gatherer := metrics.NewMultiGatherer()
+	gatherer := metrics.NewPrefixGatherer()
 	reg, m, err := newMetrics()
 	require.NoError(err)
 	vm.metrics = m
 	require.NoError(gatherer.Register("hypersdk", reg))
-	require.NoError(vm.snowCtx.Metrics.Register(gatherer))
+	require.NoError(vm.snowCtx.Metrics.Register("", gatherer))
```

Comment:
You may want to put a TODO to revisit all of this metric creation flow. I'm sure there is a cleaner way to do what I attempted to do here.

Reply:
Adding